agent_app_sample_code/agents/function_calling_agent/function_calling_agent_mlflow_sdk.py

```diff
@@ -87,9 +71,7 @@ def __call__(self, query: str) -> str:
             )
             context += formatted_chunk
 
-        resulting_prompt = self.config.get("prompt_template").format(context=context)
-
-        return resulting_prompt
+        return context.strip()
```

Comment:
I couldn't make a comment on line 79, but class Document can be replaced with: https://mlflow.org/docs/latest/python_api/mlflow.entities.html#mlflow.entities.Document not